.circleci/config.yml

```diff
@@ -1,31 +1,53 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.0
+  plugin-ci: mattermost/plugin-ci@volatile
```

Comment:
According to https://circleci.com/docs/2.0/orbs-faq/ we should not be using volatile -- can we use a version?

Reply:
Almost all other plugins use `volatile`, because that allow us to update the CI config for all plugins without having to modify them all. I'm 2/5 that we should stick with it here for consistency and easier updates-

Reply:
Sure, that makes sense.